hdl21/prefix.py

```diff
@@ -15,10 +15,14 @@
 
 """
 
+import math
```

Comment:
is this import used?

Reply:
No, it isn't, I can remove in next commit.